docs/topic_guides/choosing_comparators.ipynb

```diff
@@ -0,0 +1,1249 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>iRchard</td>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>0.952</td>\n",
+       "      <td>0.952</td>\n",
+       "      <td>1.000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Richadr</td>\n",
+       "      <td>2</td>\n",
+       "      <td>1</td>\n",
+       "      <td>0.952</td>\n",
+       "      <td>0.971</td>\n",
+       "      <td>1.000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>5</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Rich</td>\n",
+       "      <td>3</td>\n",
+       "      <td>3</td>\n",
+       "      <td>0.857</td>\n",
+       "      <td>0.914</td>\n",
+       "      <td>0.571</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Rick</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>0.726</td>\n",
+       "      <td>0.808</td>\n",
+       "      <td>0.375</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Ricky</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>0.676</td>\n",
+       "      <td>0.676</td>\n",
+       "      <td>0.333</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Dick</td>\n",
+       "      <td>5</td>\n",
+       "      <td>5</td>\n",
+       "      <td>0.595</td>\n",
+       "      <td>0.595</td>\n",
+       "      <td>0.222</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Rico</td>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>0.726</td>\n",
+       "      <td>0.808</td>\n",
+       "      <td>0.375</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>10</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Rachael</td>\n",
+       "      <td>3</td>\n",
+       "      <td>3</td>\n",
+       "      <td>0.714</td>\n",
+       "      <td>0.743</td>\n",
+       "      <td>0.444</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>11</th>\n",
+       "      <td>Richard</td>\n",
+       "      <td>Stephen</td>\n",
+       "      <td>7</td>\n",
+       "      <td>7</td>\n",
+       "      <td>0.429</td>\n",
+       "      <td>0.429</td>\n",
+       "      <td>0.083</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "    string1  string2  levenshtein_distance  damerau_levenshtein_distance  \\\n",
```

Comment:
This table is wider than everything else on the page and overlaps with the table of content. Is it possible to make it so the user scrolls?

Reply:
Have tried a couple of ways around this but no success